### DISCUSSION TITLE
Can we use version 0.4.12 for Production System?

### DISCUSSION THREAD

**Discussion by twiga2013** [microsoft/PowerPlatform-DataverseServiceClient#127] [General]
Hi,
Can we use version 0.4.12 in our Dynamics implementations? Will Microsoft support if any issues arises?

I want to use Azure Functions with .NET Core 3.1 instead of going back to V1 and use XRM Tooling SDK.

**Comment by MattB-msft** [comment]
We use it for many production scenarios now :)   including the PAC CLI, ADO and Git Hub tasks, in addition to several of our cloud services. 
We are supporting this currently though this channel ( git ) and will be brining support onboard as we begin to work though documentation migration from CrmServiceClient.
